src/test/integration/IntegrationChecks.t.sol

```diff
@@ -245,8 +283,8 @@ contract IntegrationCheckUtils is IntegrationBase {
             staker, strategies, shares, "staker should expected shares in each strategy after depositing"
         );
 
-        if (delegationManager.isDelegated(address(staker))) {
-            User operator = User(payable(delegationManager.delegatedTo(address(staker))));
+        if (delegationManager().isDelegated(address(staker))) {
```

Comment:
Just noting, I feel like making these functions (instead of variables) is a slight downgrade for brevity. But I'm fine with it atm...